model_servers/llamacpp_python/Makefile

```diff
@@ -19,9 +19,11 @@ all: build download-model-granite run
 build-cuda:
 	"${CONTAINER_TOOL}"  build --squash-all -t $(CUDA_IMAGE) . -f cuda/Containerfile
 
-.PHONY: build-vulkan
-build-vulkan:
-	"${CONTAINER_TOOL}"  build --squash-all -t $(VULKAN_IMAGE) . -f vulkan/Containerfile
+.PHONY: build-vulkan_amd64 build-vulkan_arm64
+build-vulkan_amd64:
```

Comment:
nit: Why switch from - to _?

build-vulkin-amd64?
Looks better.

Reply:
Updated 👍